src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,122 @@
+@import "./utils/variables";
+
+.catalog {
```

Comment:
checklist
<img width="666" alt="image" src="https://user-images.githubusercontent.com/102411130/213120585-43011814-1f13-4968-94ee-30cdce5cceb3.png">

`catalog.scss` for `catalog` bem block, `card.scss` for `card`